includes/definitions/discovery/arubaos-cx.yaml

```diff
@@ -35,3 +53,30 @@ modules:
                     states:
                         - { value: 1, descr: OK, graph: 1, generic: 0 }
                         - { value: 3, descr: Not OK, graph: 1, generic: 2 }
+                -
+                    oid: arubaWiredPowerSupplyTable
+                    num_oid: '.1.3.6.1.4.1.47196.4.1.1.3.11.2.1.1.11.{{ $index }}'
+                    descr: 'Power Supply {{ $arubaWiredPSUName }}'
+                    value: arubaWiredPSUStateEnum
+                    states:
+                        - { value: 1, descr: ok, graph: 1, generic: 0 }
+                        - { value: 2, descr: Absent, graph: 1, generic: 2 }
+                        - { value: 3, descr: Fault Input, graph: 1, generic: 2 }
+                        - { value: 4, descr: Fault Output, graph: 1, generic: 2 }
+                        - { value: 5, descr: Fault POE, graph: 1, generic: 2 }
+                        - { value: 6, descr: Fault No Recovery, graph: 1, generic: 2 }
+                        - { value: 7, descr: alert, graph: 1, generic: 2 }
+                        - { value: 8, descr: unknown, graph: 1, generic: 2 }
+                        - { value: 9, descr: unsuppored, graph: 1, generic: 2 }
```

Comment:
Does this need to be generic 3 if it's not really a fault (or generic 1 for it to be ok).

Reply:
> Does this need to be generic 3 if it's not really a fault (or generic 1 for it to be ok).

Personally, I prefer generic 3 because it's not really "ok". Or are there community recommendations for such cases?

Reply:
3 is probably ok but you've got it as 2 currently.